### ISSUE TITLE
Trying to change most fetish options doesn't update the GUI.

### ISSUE THREAD

**Issue by zR1OQicz** [Innoxia/liliths-throne-public#1401]
**Describe the bug**
Trying to change most fetish options doesn't update the GUI. A NullPointerException is raised.

**Expected behavior**
The options can be toggled normally. The GUI is updated on click.

**To Reproduce**
1. Start the game v0.3.9.4. The main screen appears.
2. Click "Content Options". The option screen appears.
3. Click "Sex & Fetishes". The Fetish selection appears.
4. Try toggling e.g. the "Incest" option. Nothing happens.
5. Toggle a working option (e.g. "Sadistic Sex"). The display is updated, the previously non-working option is changed as well.

**Attach files**
error.log:
```
Version: 0.3.9.4
Java: 14.0.1
java.lang.NullPointerException
	at com.lilithsthrone.controller.MainControllerInitMethod.lambda$createToggleListener$415(MainControllerInitMethod.java:7054)
	at javafx.web/com.sun.webkit.dom.EventListenerImpl.fwkHandleEvent(EventListenerImpl.java:129)
	at javafx.web/com.sun.webkit.WebPage.twkProcessMouseEvent(Native Method)
	at javafx.web/com.sun.webkit.WebPage.dispatchMouseEvent(WebPage.java:809)
	at javafx.web/javafx.scene.web.WebView.processMouseEvent(WebView.java:1009)
	at javafx.web/javafx.scene.web.WebView.lambda$registerEventHandlers$3(WebView.java:1132)
	at javafx.base/com.sun.javafx.event.CompositeEventHandler$NormalEventHandlerRecord.handleBubblingEvent(CompositeEventHandler.java:247)
	at javafx.base/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:80)
	at javafx.base/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:234)
	at javafx.base/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:191)
	at javafx.base/com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(CompositeEventDispatcher.java:59)
	at javafx.base/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
	at javafx.base/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at javafx.base/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at javafx.base/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at javafx.base/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at javafx.base/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at javafx.base/com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
	at javafx.base/com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:54)
	at javafx.base/javafx.event.Event.fireEvent(Event.java:198)
	at javafx.graphics/javafx.scene.Scene$MouseHandler.process(Scene.java:3890)
	at javafx.graphics/javafx.scene.Scene.processMouseEvent(Scene.java:1885)
	at javafx.graphics/javafx.scene.Scene$ScenePeerListener.mouseEvent(Scene.java:2618)
	at javafx.graphics/com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:409)
	at javafx.graphics/com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:299)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at javafx.graphics/com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$2(GlassViewEventHandler.java:447)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(QuantumToolkit.java:412)
	at javafx.graphics/com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(GlassViewEventHandler.java:446)
	at javafx.graphics/com.sun.glass.ui.View.handleMouseEvent(View.java:556)
	at javafx.graphics/com.sun.glass.ui.View.notifyMouse(View.java:942)
	at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277)
	at java.base/java.lang.Thread.run(Thread.java:832)
```

**Additional information:**
- Game version v0.3.9.4
- OS: SuSE Linux 64-bit
- self-compiled version from github commit 1532da80b1980c40a0605d62ff6489afe9637f82
- java --version
  openjdk 14.0.1 2020-04-14
  OpenJDK Runtime Environment (build 14.0.1+7-suse-1.3-x8664)
  OpenJDK 64-Bit Server VM (build 14.0.1+7-suse-1.3-x8664, mixed mode)

**Comment by AceXP** [comment]
This error occurs because there is no game started yet. The toggles are set, but then a recalculation is attempted which fails with the NullPointer.
Once a game is started everything will be fine. The toggle state is remembered and everything will be calculated again, so there is no permanent effect.

**Comment by zR1OQicz** [comment]
I can confirm that the options work when toggled after a game is running. I still consider this a bug, because the options can be accessed from the main screen and should work even if no game has been loaded.

**Comment by AceXP** [comment]
Yes, it's a bug. I just wanted to point out that it is not very critical. Anyway the fix is easy so hopefully it will be fixed in a future version.

**Comment by zR1OQicz** [comment]
Verified fixed in v0.3.9.8
